vagrant-service-manager.spec

```diff
@@ -69,6 +69,7 @@ popd
 %{vagrant_plugin_libdir}
 %exclude %{vagrant_plugin_cache}
 %{vagrant_plugin_spec}
+%{vagrant_plugin_instdir}/plugins/guests/
```

Comment:
As per https://fedoraproject.org/wiki/PackagingDrafts/Vagrant#Macros %{vagrant_plugin_instdir} should contain actual content of the plugin like Gemfile, Rakefile, tests ..etc. Plugin gem should be part of %{vagrant_plugin_dir} and code content should be part of %{vagrant_plugin_libdir}